src/test/java/org/opensearch/neuralsearch/query/NeuralSparseQueryBuilderTests.java

```diff
@@ -88,7 +93,6 @@ public void testFromXContent_whenBuiltWithOptionals_thenBuildSuccessfully() {
               "VECTOR_FIELD": {
                 "query_text": "string",
                 "model_id": "string",
-                "max_token_score": 123.0,
```

Comment:
can we keep a unit test where we are providing the deprecated field and no impact on queries are happening.

Reply:
I think we already have this.